airbyte-integrations/connectors/source-zenloop/source_zenloop/manifest.yaml

```diff
@@ -113,11 +115,10 @@ definitions:
       requester:
         $ref: "#/definitions/incremental_base_stream/retriever/requester"
         path: "{{ 'surveys/' + stream_slice.id + '/answers' }}"
-      stream_slicer:
-        type: CartesianProductStreamSlicer
-        stream_slicers:
-          - "#/definitions/surveys_slicer"
-          - "#/definitions/date_slicer"
+      incremental_sync:
```

Comment:
Do we still need the `incremental_sync`? If anything it should be one level above in the stream instead of the retriever, but we're also already defining it it in the `base_incremental_sync` block above too. Same for the one below

Reply:
You are right. Both stream are referencing `$ref: "#/definitions/incremental_base_stream"`and this is just ignored by the parser and therefore is useless. I'll be cleaning this. Thanks!